lib/policies/index.js

```diff
@@ -49,14 +49,25 @@ schemas.register('internal', 'baseAuth', {
   required: ['passThrough']
 });
 
-const policyNames = fs
-  .readdirSync(path.resolve(__dirname))
-  .filter(dir => fs.lstatSync(path.resolve(__dirname, dir)).isDirectory());
-
-policyNames.forEach((name) => {
-  const policy = require(path.resolve(__dirname, name));
-  policies[name] = Object.assign(policy, { name });
-  register(policy);
-});
+const load = (policiesToBeLoaded) => {
+  const policyNames = fs
+    .readdirSync(path.resolve(__dirname))
+    .filter(dir => fs.lstatSync(path.resolve(__dirname, dir)).isDirectory());
+
+  const policiesWithoutPlugins = policiesToBeLoaded.filter((name) => policyNames.includes(name));
```

Comment:
Let's use a better name

```suggestion
  const corePolicies = policiesToBeLoaded.filter(name => policyNames.includes(name));
```